src/cli/commands/test/mod.rs

```diff
@@ -0,0 +1,164 @@
+#[cfg(test)]
+mod tests;
+
+use regex::Regex;
+
+use std::fs;
+use serde_json::Value;
+use std::{collections::HashMap, fmt::Display, io::Write, path::PathBuf, str::from_utf8};
+use cairo_rs::{
+	cairo_run::cairo_run,
+	hint_processor::{
+		builtin_hint_processor::{
+			builtin_hint_processor_definition::{BuiltinHintProcessor, HintFunc},
+			hint_utils::get_integer_from_var_name,
+		},
+		hint_processor_definition::HintReference,
+		proxies::{exec_scopes_proxy::ExecutionScopesProxy, vm_proxy::VMProxy},
+	},
+	serde::deserialize_program::ApTracking,
+	vm::{errors::{vm_errors::VirtualMachineError}},
+};
+use clap::{Args, ValueHint};
+use log::error;
+use serde::Serialize;
+
+use super::CommandExecution;
+use super::list::{path_is_valid_directory, ListArgs};
+
+
+use crate::cli::commands::execute::compile::compile;
+
+#[derive(Args, Debug)]
+pub struct TestArgs {
+	/// Path to a cairo directory
+	#[clap(short, long, value_hint=ValueHint::DirPath, value_parser=path_is_valid_directory)]
+	pub root: PathBuf,
+
+
+
+
+}
+
+fn list_test_entrypoints(compiled_path: &PathBuf) -> Result<Vec<String>, String> {
+	let re = Regex::new(r"__main__.(test_\w+)").map_err(|_op| format!("Regex failed"))?;
```

Comment:
Your regex is known at compile time. So you know it won't fail to create a regex from your string. So you un just `unwrap` or `expect` (which is better) there. Instead of forwarding a error that you know will never occur

Reply:
done!